azext_iot/central/_help.py

```diff
@@ -319,6 +320,92 @@ def _load_central_api_token_help():
     """
 
 
+def _load_central_cde_help():
+    helps[
+        "iot central app cde"
+    ] = """
+        type: group
+        short-summary: Create and Manage Continuous Data Export (CDE) for an application.
```

Comment:
With current release of CDE V2 we are going to start calling this CDE , CDE (legacy)